### ISSUE TITLE
feature request: allow simple pen gestures

### ISSUE THREAD

**Issue by ShBky** [TransposonY/GestureSign#94]
Right now, you can only make pen gestures work, if drawn with the right click button held down, or written with eraser side. Please allow the user to create gestures, that activate, when you draw the gesture regularly with the tip side.

**Comment by seflerZ** [comment]
Isn't it will collision with regular drawing applications with pen tip side such as OneNote?

**Comment by asampal** [comment]
Of course some of these gestures could get triggered by regular strokes if they were very basic/simple, but I think the idea is that they would be specific enough so that they would not trigger by accident.

**Comment by seflerZ** [comment]
@asampal But it won't be easy. Suppose you hava such gesture enabled and specific. How to handle the ink already on the screen? The GestureSign doesn't have permission to erase ink on other applications I thought.

**Comment by asampal** [comment]
If GestureSign acted as a filter with a sliding window of, say, 1.5 seconds, for all strokes as they are drawn, then if it detected what looked like a gesture (possibly made of multiple strokes) within that window, it would trigger the associated action.

**Comment by seflerZ** [comment]
On that way, it will cause bad experience (strong latency) on original drawing apps. The drawing apps are designed to accept any strokes as fast as possible.

**Comment by asampal** [comment]
You're right - back when I was considering this, I had in mind holding down two fingers while doing the gesture. I suppose that might be considered less bothersome by some people compared to doing them with a right-click (not easy with a stylus), or with the eraser.

**Comment by seflerZ** [comment]
@asampal Adding two finger holding down sounds cool. Right-click is not easy indeed.
